ansible/roles/web/templates/parameters.yml.j2

```diff
@@ -49,3 +49,4 @@ parameters:
     # Algolia Configuration
     env(ALGOLIA_APP_ID):  '{{ site_algolia_app_id }}'
     env(ALGOLIA_API_KEY): '{{ site_algolia_api_key }}'
+    ALGOLIA_SEARCH_KEY: '{{ site_algolia_search_key }}'
```

Comment:
No `env()` needed on that one? Why? Can we keep the two consistent?

Reply:
env() is used to substitute for a missing env var, where it is used in getenv. This is supposed to be used directly on the front end.